packages/terra-application/docs/README.md

```diff
@@ -0,0 +1,285 @@
+# Terra Application
+
+The `terra-application` package provides a baseline of functionality for applications built with Terra components.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-application`
+
+## Usage
+
+The `terra-application` package should be used both by applications and libraries to define a baseline of expected features and functionality for their components.
+
+For application packages:
+
+1. `terra-application` must be specified as a dependency.
+2. A single `<Application />` component must be rendered as high as possible in the component tree.
+3. Components provided by `terra-application` should be consumed solely through `terra-application` and not through their own separate packages.
+
```

Comment:
Might be good to have a do / don't example of how we expect consumers to import packages provided by terra-application.

Reply:
I've expanded this section pretty heavily. Please take a look and see what you think.

Reply:
Looks good to me.